### PR TITLE
Cherry-pick #9581 into release/0.42

### PR DESCRIPTION
**Pull request by lukelee-sl**
**Description**:
Increase the number of KV pairs allowed for contract storage by a factor of 100 x

### CODE REVIEW

**Review by netopyr — COMMENTED** [review]
I wonder if it makes sense to update the code if we do not plan to deploy it. It may be confusing in the future if the file for the overridden properties contains the default values.

**Review by Nana-EC — DISMISSED** [review]

**Comment by lukelee-sl** [comment]
> I wonder if it makes sense to update the code if we do not plan to deploy it. It may be confusing in the future if the file for the overridden properties contains the default values.

For clarity, are you suggesting that I remove all changes to the *.java files or something else?

File: hedera-node/hedera-app/src/test/resources/bootstrap.properties [hunk 1]
```diff
@@ -78,8 +78,8 @@ contracts.keys.legacyActivations=1058134by[1062784]
 contracts.knownBlockHash=
 contracts.localCall.estRetBytes=32
 contracts.maxGasPerSec=15000000
-contracts.maxKvPairs.aggregate=500_000_000
-contracts.maxKvPairs.individual=163_840
+contracts.maxKvPairs.aggregate=500000000
```
Comment (poulok) [inline hedera-node/hedera-app/src/test/resources/bootstrap.properties:81]: I'm not sure why the underscores were removed. They are helpful when verifying the correct number is set.

Comment (lukelee-sl) [reply hedera-node/hedera-app/src/test/resources/bootstrap.properties:81]: The underscores were causing some of the HAPI tests to fail in the modularized code with `NumberFormatException`.  Apparently, the logic for parsing the config values with the underscores has not been implemented?  We probably went overboard on removing them but we wanted to be sure that the tests would pass.

Comment (Nana-EC) [reply hedera-node/hedera-app/src/test/resources/bootstrap.properties]: @lukelee-sl can we open a ticket to fix that in modularization. 
FYI @hendrikebbers

**Review by Nana-EC — APPROVED** [review]
LG
Values are update and overrides for TN and MN envs are added

File: hedera-node/hedera-app/src/test/resources/bootstrap.properties (hunk 1 above)
Comment (lukelee-sl) [reply hedera-node/hedera-app/src/test/resources/bootstrap.properties]: looks like an issue was already created by @david-bakin-sl 
https://github.com/hashgraph/hedera-services/issues/9589

**Review by rbair23 — APPROVED** [review]

**Review by tinker-michaelj — APPROVED** [review]
LGTM!

**Review by povolev15 — APPROVED** [review]

**Pull request merged**